docs/reference/mouse_bindings.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Mouse bindings
+---
+
+The mousemap files (which have the extension `.sublime-mousemap`)
+control what commands are executed when a user performs an action
+with a mouse e.g. clicking a mouse button, scrolling the scroll
+wheel etc.
+
```

Comment:
The semantic linefeeds in this paragraph are a bit off.

Please also mention that mousemaps use JSON.

Reply:
TBH, I have not really understood the importance of semantic line feeds even after reading the linked URL in the CONTRIBUTING.md. If you can tell how the paragraph needs to be broken down or reworded, I'll do it.

Reply:
I'll do a manual pass on this later that you can review.